app/models/page_content_register.rb

```diff
@@ -17,7 +19,10 @@ def new_blocks_attributes
                         else
                           @new_blocks_attributes
                         end
-    return @new_blocks_attributes if home_page? || footer?
+
+    if home_page? || footer?
+      return blocks_attributes + srcset_blocks(blk_attrs: blocks_attributes)
```

Comment:
Reading only this line, makes an assumption of "something is sum/adding things to the blocks attributes hash". But reading what the srcset_blocks method does, on my understanding, it is "changing the identifier prefix and add the styles of the block content of the block attributes if is raw_tile_\d_image". So, is this sum necessary? And do you think *srcset_blocks* is a good name to clarity on what the method is doing?